src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.html

```diff
@@ -0,0 +1,12 @@
+<div #tooltipHoverElement (mouseenter)="emitNewHoverMouseEvent($event)" (mousemove)="emitNewHoverMouseEvent($event)" (mouseleave)="emitNewHoverMouseEvent()" class="c-hover-scrub-hover-element"></div>
+
+<sofie-tooltip *ngIf="tooltipHoverElement" [tooltipElementHoverMouseEventObservable]="mouseHoverEventSubject.asObservable()">
+  <sofie-video-hover-scrub
+    [fileName]="fileName"
+    [tooltipHoverElementWidth]="tooltipHoverElement.clientWidth"
+    [type]="type"
+    [pieceLifespan]="pieceLifespan"
+    [tooltipHoverMouseEventObservable]="mouseHoverEventSubject.asObservable()"
+    [isMediaUnavailable]="isMediaUnavailable">
+  </sofie-video-hover-scrub>
```

Comment:
The VideoHoverScrub component should only show the actual video hoverscrub.
Meaning it should _not_ show the Piece name, the full duration and the Piece lifespan. I.e. everything on the "right side" of this picture:
![image](https://github.com/tv2/ng-sofie/assets/23049647/625a8312-fbe9-417e-b652-ceb6fbc0bc81)

All that information should be shown in "Tooltip part" of PieceTooltipComponent. This right here is the "Hoverscrub part".

Reply:
@LindvedKrvang if we move right part of the logic to piece-tooltip we need to make same copy for mini-shelf-tooltip because now this 2 tooltips don't have any differences. We can create one extra component and separate the logic between left (video) and right (text) part of the video tooltip. Is that ok?

Reply:
They don't have any differences right now, but they will as soon SOF-1636 is commenced. 
The "right side" of the tooltip is out of scope for your current task.

No need to create an extra component for now. Your task is only to do the "left side" which you already did. The "left side" just need to have this extra data, which doesn't belong in it, removed.

Reply:
Done